### PR TITLE
Remove director internal graphQL API. Change OTT for csr issuing

### PR DESCRIPTION
**Pull request by pankrator**
**Remove director internal graphQL API. Change OTT for csr issuing**

Changes proposed in this pull request:
- Remove director internal graphQL API
- For connector re-issue of OTT use the regular public graphQL API (requestOneTimeTokenFor...)
- Remove magic from application resolver for auths entity
- Return system auths as is with one time token if any and allow clients for reason for the returned tokens
- Add `certCommonName` property to system auth in database to show that there is associated a system auth with an issued connector certificate

- [x] Implementation
- [x] Unit tests
- [x] Integration tests
- [x] `chart/compass/values.yaml` is updated

### CODE REVIEW

File: components/connector/internal/authentication/context_middleware.go [hunk 1]
```diff
@@ -15,6 +24,35 @@ func NewAuthenticationContextMiddleware() *authContextMiddleware {
 
 func (acm *authContextMiddleware) PropagateAuthentication(handler http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		ctx := r.Context()
+		authorizationHeader := r.Header.Get("Authorization")
+		if authorizationHeader != "" {
+			jwtTokenPayload, err := base64.RawStdEncoding.DecodeString(strings.Split(authorizationHeader[7:], ".")[1])
```
[the remark this answers is not in the document]
Comment (pankrator) [reply components/connector/internal/authentication/context_middleware.go]: it worked :)

File: components/director/internal/domain/onetimetoken/service.go [hunk 2]
```diff
@@ -330,9 +351,9 @@ func (s *service) IsTokenValid(systemAuth *model.SystemAuth) (bool, error) {
 		return false, errors.Errorf("One Time Token for system auth id %s has been used", systemAuth.ID)
 	}
 
-	expirationTime, err := s.getExpirationTimeForToken(systemAuth)
+	expirationTime, err := s.getExpirationDurationForToken(systemAuth.Value.OneTimeToken.Type)
 	if err != nil {
-		return false, err
+		return false, errors.Wrapf(err, "one-time token for system auth id %s has no valid expiration type", systemAuth.ID)
```
Comment (krasish) [inline components/director/internal/domain/onetimetoken/service.go:356]: expiration type? Not sure what that means.

**Pull request merged**